lib/action_policy.rb

```diff
@@ -21,6 +21,7 @@ def initialize(target)
   require "action_policy/base"
   require "action_policy/lookup_chain"
   require "action_policy/behaviour"
+  require "action_policy/i18n"
```

Comment:
What about requiring this file only if `I18n` is defined and avoid runtime checks?

Reply:
That would also allow us to run tests altogether

Reply:
Done. It helped for the missing i18n spec, but I still have to run i18n specs in s separate task because otherwise i18n would be required when missing spec starts running

Reply:
We don't need to test the case when `i18n` is missing at all, so we can assume that's it's loaded (and load in `test_helper.rb`)

Reply:
✔️